### PR TITLE
Create adminhtml layout and templates

### PR DESCRIPTION
**Pull request by adam-paterson**
adminhtml layout and template directories aren't created. I also think it's important to show the module structure that is created. Although I agree with it some people might argue what the perfect  structure is.

### CODE REVIEW

**Comment by AydinHassan** [comment]
Nice. I reckon it might be better with an actual real module name instead of vendor/namespace in the example structure. What do you think?

**Comment by adam-paterson** [comment]
Yeah agreed, I did have AydinHassan/Parrot as an example. I'll put that back in

**Comment by adam-paterson** [comment]
Updated

File: build.php [hunk 1]
```diff
@@ -148,15 +148,31 @@
...
+    ];
+
+    $templateDirs = [
+        "frontend" => 'app/design/frontend/base/default/template/' . strtolower($vendor) . '/' . strtolower($nameSpace),
+        "adminhtml" => 'app/design/adminhtml/base/default/template/' . strtolower($vendor) . '/' . strtolower($nameSpace),
+    ];
+
+    foreach ($layoutDirs as $layoutDir) {
+        mkdir($layoutDir, 0777, true);
+        unlink('app/design/frontend/base/default/layout/.gitkeep');
```
Comment (AydinHassan) [inline build.php]: These unlinks need to be outside the loop as the second call emits a warning as it no longer exists

File: build.php [hunk 2]
```diff
@@ -148,15 +148,31 @@
...
+        unlink('app/design/adminhtml/base/default/layout/.gitkeep');
+    }
+
+    foreach ($templateDirs as $templateDir) {
+        mkdir($templateDir, 0777, true);
+        unlink('app/design/frontend/base/default/template/.gitkeep');
```
Comment (AydinHassan) [inline build.php]: Same as above

**Comment by AydinHassan** [comment]
Only just noticed you'd fixed the unlinks, if you can squash these commits I'll get this merged!

**Comment by adam-paterson** [comment]
Squashed!

**Comment by AydinHassan** [comment]
Thanks!

**Pull request merged**